generators/app/index.js

```diff
@@ -147,10 +146,47 @@ module.exports = class extends Generator {
       {}
     );
 
+    this.fs.copyTpl(
+      this.templatePath('.eslintrc'),
+      this.destinationPath('.eslintrc'),
+      {}
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('.eslintignore'),
+      this.destinationPath('.eslintignore'),
+      {}
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('.prettierrc'),
+      this.destinationPath('.prettierrc'),
+      {}
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('.gitignore'),
+      this.destinationPath('.gitignore'),
+      {}
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('tests/index.test.js'),
+      this.destinationPath('tests/index.test.js'),
+      {}
+    );
+
     this.fs.copyTpl(this.templatePath('TODO.md'), this.destinationPath('TODO.md'), {});
   }
 
   install() {
+    this.spawnCommandSync('git', ['init']);
+    this.spawnCommandSync('git', ['add', '.']);
+    this.spawnCommandSync('git', [
+      'commit',
```

Comment:
Nice idea to create an initial commit! :+1:

Reply:
Thanks 😃